app/src/features/apiClient/screens/apiClient/components/views/graphql/GraphQLClientView.tsx

```diff
@@ -516,11 +518,15 @@ const GraphQLClientView: React.FC<Props> = ({
         <div className="api-client-header-container__header">
           <div className="api-client-breadcrumb-container">
             <ApiClientBreadCrumb
-              id={record.id}
+              id={apiEntryDetails.id}
               placeholder="Untitled request"
               openInModal={openInModal}
-              name={record.name}
-              onBlur={(newName) => handleRecordNameUpdate(newName)}
+              name={apiEntryDetails?.name}
```

Comment:
Lets not create two sources of API record here. If record was being used here, lets continue using it. 

As I can see, `record.name` is not available then we can use a default value when its unavailable: `record.name ?? "Untitled request"`